taca/server_status/server_status.py

```diff
@@ -9,185 +9,186 @@
+    credentials = GCredentials(json_key['client_email'], json_key['private_key'], config['g_scope'])
+    gc = gspread.authorize(credentials)
+    # open google sheet
+    # IMPORTANT: file must be shared with the email listed in credentials
+    sheet = gc.open(config['g_sheet'])
 
-	# choose worksheet from the doc
-	worksheet = sheet.get_worksheet(1)
+    # choose worksheet from the doc
+    worksheet = sheet.get_worksheet(1)
 
-	# update cell
-	for key in data: # data is a dicitonary of dictionaries
-		cell = config['g_sheet_map'].get(key) # key = server name
-		value = data[key].get('available_percentage')
-		worksheet.update_acell(cell, value)
+    # update cell
+    for key in data: # data is a dicitonary of dictionaries
+        cell = config['g_sheet_map'].get(key) # key = server name
+        value = data[key].get('available_percentage')
+        worksheet.update_acell(cell, value)
 
 
 def update_status_db(data):
-	"""Pushed the data to status db,
-	data can be from nases or from uppmax
-	"""
-	db_config = CONFIG.get('statusdb')
-	if db_config is None:
-		logging.error("'statusdb' must be present in the config file!")
-		raise
-
-	server = "http://{username}:{password}@{url}:{port}".format(
+    """Pushed the data to status db,
+    data can be from nases or from uppmax
+    """
+    db_config = CONFIG.get('statusdb')
+    if db_config is None:
+        logging.error("'statusdb' must be present in the config file!")
+        raise
+
+    server = "http://{username}:{password}@{url}:{port}".format(
         url=db_config['url'],
         username=db_config['username'],
         password=db_config['password'],
         port=db_config['port'])
-	try:
-		couch = couchdb.Server(server)
-	except Exception, e:
-		logging.error(e.message)
-		raise
-
-	db = couch['server_status']
-	logging.info('Connection established')
-	for key in data.keys(): # data is dict of dicts
-		server = data[key] # data[key] is dictionary (the command output)
-		server['name'] = key # key is nas url or uppmax project
-		server['time'] = datetime.datetime.now().isoformat() # datetime.datetime(2015, 11, 18, 9, 54, 33, 473189) is not JSON serializable
-
-		try:
-			db.save(server)
-		except Exception, e:
-			logging.error(e.message)
-			raise
-		else:
-			logging.info('{}: Server status has been updated'.format(key))
+    try:
+        couch = couchdb.Server(server)
+    except Exception, e:
+        logging.error(e.message)
+        raise
+
+    db = couch['server_status']
+    logging.info('Connection established')
+    for key in data.keys(): # data is dict of dicts
+        server = data[key] # data[key] is dictionary (the command output)
+        server['name'] = key # key is nas url or uppmax project
+        server['time'] = datetime.datetime.now().isoformat() # datetime.datetime(2015, 11, 18, 9, 54, 33, 473189) is not JSON serializable
+        server['server_type'] = 'uppmax' if 'uppmax' in server['name'] else 'nas'
```

Comment:
seems fine I have an hard time putting together the three pull request but as long as it tested it is fine